doc/themeCustomization.md

```diff
@@ -2,76 +2,93 @@
 
 ## Create a new color theme
 
-- First, duplicate directory [/src/theme/custom/default](../src/theme/custom/default) and rename it as your wish,
-  exemple('fooDark', 'fooLight', etc.).
+- First, duplicate the directories [/src/theme/custom/cosmoDark](../src/theme/custom/cosmoDark)
+ (resp. [/src/theme/custom/cosmoLight](../src/theme/custom/cosmoLight)) and rename it fooDark (resp. fooLight) for instance.
 
-- Then, in the newly created dircetory open the matching file [/src/theme/custom/fooDark/palette.js](../src/theme/custom/fooDark/palette.js) and
-  set the colors dictionnary:
-
-```javascript
-const themeColors = {
-  primary: '#FFAD38',
-  primaryVariant: '#A16612',
-  secondary: '#466180',
-  background: '#191919',
-  backgroundVariant: '#1F1F1F',
-  surface: '#2E2E2E',
-  error: '#FF667F',
-  warning: '#FB8C00',
-  success: '#19E152',
-  info: '#1E88E5',
-  white: '#FFFFFF',
-  black: '#000000',
-  grey: '#999A9D',
-  backgroundSignInPage: '#2f363b',
-  backgroundSignInButton: '#2F2F2F',
-};
-```
-
-- Then you can recompose the correspondance between color and display elements:
+- Then, in the newly created directory, open [/src/theme/custom/fooDark/palette.js](../src/theme/custom/fooDark/palette.js)
+ (resp. [/src/theme/custom/fooLight/palette.js](../src/theme/custom/fooLight/palette.js))
+ and set the colors dictionary as follow (colors are given as an example):
 
 ```javascript
 export default {
-  type: 'light',
-  white: themeColors.white,
-  black: themeColors.black,
+  mode: 'dark',
```

Comment:
No it's not, and this is something I am currently fixing in my last theme's branch, so I updated the doc in advance. Both seem to work, but the theme on MUI website uses `mode` instead of `type`, so maybe `type` is only deprecated.